CollapseLauncher/Classes/InstallManagement/Base/InstallManagerBase.cs

```diff
@@ -65,6 +65,7 @@
 using ZipArchive = SharpCompress.Archives.Zip.ZipArchive;
 using ZipArchiveEntry = SharpCompress.Archives.Zip.ZipArchiveEntry;
 // ReSharper disable SwitchStatementMissingSomeEnumCasesNoDefault
+#pragma warning disable CS8777 // Parameter must have a non-null value when exiting.
```

Comment:
I dont think I prefer a global pragma warning this early on the file, if you must do it please do it on the specific piece of code then restore it